man/systemd-sysext.xml

```diff
@@ -139,6 +139,7 @@
     architecture reported by <citerefentry><refentrytitle>uname</refentrytitle><manvolnum>2</manvolnum></citerefentry>
     but the used architecture identifiers are the same as for <varname>ConditionArchitecture=</varname>
     described in <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
+    <varname>EXTENSION_RELOAD_MANAGER=</varname> can be set to 1 if the extension requires a manager reload.
```

Comment:
This needs to warn that doing so might cause race conditions and break booting the system, so it should be done with care and only when really unavoidable, and point to https://systemd.io/PORTABLE_SERVICES/ as a better and safer alternative

Reply:
I think suggesting portable services here makes sense, with an explanation why they are the preferable way to ship services, but I wouldn't use too harsh words, and avoid vague wording such as "race conditions and break booting the system". Yes, daemon reloads during bootup suck, but if limited in scope they should be well managable.

Reply:
It can be mentioned in a nicer way, but I think it should be mentioned, because when it does become an issue, it is incredibly painful to figure out - ask me how I know :-)

Reply:
Please share some details even if not for the manpage ;)
I hope we can skip the reload during boot when we implement the overlay mounting from the initrd. This could be done by letting the service survive the transition or with explicit checks where we would have to track the loaded extensions and the service start could then see whether it would only load the same things again and skip it.